### ISSUE TITLE
Remote services - what features do they offer

### ISSUE THREAD

**Issue by j-i-l** [t4d-gmbh/git-and-its-remotes#4]
https://github.com/t4d-gmbh/using-git-in-academia/blob/ffb0b0ca004ac04f2f4491d15d01b5ce4484418b/README.md?plain=1#L44

> - Be able to tell what features and tools belong to git and what belong to remote services
---

# To Do

The need the following sections:

- `Remote features`
  - Present `Issues`, `Merge/Pull requests`, `Milestones` as the 3 basic elements
    - `issues` (separate slide)
      - what they are made of (title, description, assignee, labels, start/due date)
      - show tracking and comment function
      - not just for issues
      - but if it is a bug: how to report it in a useful manner
    - `milestones` (separate slide)
      - basically a collection of issues with start and stop dates
    - `merge/pull requests` (separate slide)
      - what they are made of (title, description, assignee, reviewer, ..., reference to branch, overview of branch history, overview of changes)
      - mention the review functionality
   - Briefly mention `releases`,  `packges`/`registry`, `issue boards`, `snippets` (are these git tracked?), `wiki`, `static pages`.
     - take home message here is: lots of features, go and check their description if you are interested. They all have very little to do with git itself.
- `How git and remote features relate`
  - Basically describe _Issue > new branch > merge/pull request > updated healthy reference branch_
